src/app/projects/project-list/project-list.component.scss

```diff
@@ -254,7 +254,72 @@
   }
 }
 
-.results {
-  color: white;
-  font-weight: 700;
+.hero-banner {
```

Comment:
Thanks a lot for all the little touch ups including making the banner have better contrast!